index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Welcome to Potionfy</title>
```

Comment:
Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

Reply:
Sure can do!